datadog/dogshell/screenboard.py

```diff
@@ -199,8 +199,8 @@ def _update(cls, args):
             raise Exception('bad json parameter')
 
         res = api.Screenboard.update(
-            args.screenboard_id, title=args.title, description=args.description,
-            graphs=graphs, template_variables=args.template_variables,
+            args.screenboard_id, board_title=args.title, description=args.description,
+            widgets=graphs, template_variables=args.template_variables,
```

Comment:
Was it not working before ?

Reply:
Nope, not from what I can tell. `Graphs` are a timeboard thing. `Widgets` are for screenboards (same for title vs board_title) - https://docs.datadoghq.com/graphing/guide/screenboard-api-doc/?tab=python#arguments